sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-saveAssessmentData.e2e-spec.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  createE2EDataSources,
+  E2EDataSources,
+  saveFakeApplication,
+} from "@sims/test-utils";
+
+import { ICustomHeaders, IOutputVariables } from "zeebe-node";
+import {
+  createFakeWorkerJob,
+  FAKE_WORKER_JOB_RESULT_PROPERTY,
+  MockedZeebeJobResult,
+} from "../../../../../test/utils/worker-job-mock";
+import { createTestingAppModule } from "../../../../../test/helpers";
+import { AssessmentController } from "../../assessment.controller";
+import { SaveAssessmentDataJobInDTO } from "../../assessment.dto";
+import { createFakeSaveAssessmentDataPayload } from "./save-assessment-data-factory";
+
+describe("AssessmentController(e2e)-saveAssessmentData", () => {
```

Comment:
shouldn't there be a `client` type in the file name and here, example, 
![image](https://github.com/bcgov/SIMS/assets/77353155/3d45f2b4-00dc-4bab-a0ab-a697e163f309)

Reply:
Why?

Reply:
nvrmind, wrong comment